_includes/pbc/all/install-features/202304.0/install-the-spryker-core-back-office-feature.md

```diff
@@ -0,0 +1,607 @@
+{% info_block errorBox %}
+
+This feature integration guide expects the basic feature to be in place.
+The current feature integration guide adds the following functionalities:
+* Translation
+* Security
+* OAuth 2.0/Open ID Connect Support for Zed login
+
+{% endinfo_block %}
+
+## Prerequisites
+
+Ensure that the related features are installed:
+
+| NAME                                                                                                             | VERSION          |
+|------------------------------------------------------------------------------------------------------------------|------------------|
+| [Spryker Core](/docs/scos/dev/feature-integration-guides/{{site.version}}/spryker-core-feature-integration.html) | {{site.version}} |
+
+
+## 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker-feature/spryker-core-back-office:"{{site.version}}" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the following modules have been installed:
+
+| MODULE                     | EXPECTED DIRECTORY                           |
+|----------------------------|----------------------------------------------|
+| MessengerExtension         | vendor/spryker/messenger-extension           |
+| SecurityGui                | vendor/spryker/security-gui                  |
+| SecurityOauthUser          | vendor/sprkyer/security-oauth-user           |
+| Translator                 | vendor/spryker/translator                    |
+| UserLocale                 | vendor/spryker/user-locale                   |
+| UserLocaleGui              | vendor/spryker/user-locale-gui               |
+| UserPasswordReset          | vendor/spryker/user-password-reset           |
+| UserPasswordResetExtension | vendor/spryker/user-password-reset-extension |
+| UserPasswordResetMail      | vendor/spryker/user-password-reset-mail      |
```

Comment:
UserBackendApi is missing.

Reply:
It's not full IG, it only describes 
* Translation
* Security
* OAuth 2.0/Open ID Connect Support for Zed login

UserBackendApi is not a part of any of these, so I decided not to add it.